cli/tests/integration/lint_tests.rs

```diff
@@ -114,3 +116,66 @@ itest!(lint_with_malformed_config2 {
   output: "lint/with_malformed_config2.out",
   exit_code: 1,
 });
+
+#[test]
+fn lint_with_glob_config() {
+  let context = TestContextBuilder::new().cwd("lint").build();
+
+  let cmd_output = context
+    .new_command()
+    .args("lint --config deno.glob.json")
+    .run();
+
+  cmd_output.assert_exit_code(1);
+
+  let output = cmd_output.combined_output();
+  assert!(output.contains("glob/nested/fizz/fizz.ts:1:10"));
```

Comment:
It migth be beneficial to use the `assert_contains` macro as it provides better information on failure.